### DISCUSSION TITLE
Help me help you. I want to donate but don't use PayPal. Thanks

### DISCUSSION THREAD

**Discussion by RainerRocks** [Rem0o/FanControl.Releases#1989] [Q&A] [answered]
Hi,
 I want to donate but I don't use PayPal and not giving my CC to them as a guest.

I would like to send a $50 gift card via Amazon if possible, let me know. Thanks.

**Comment by Rem0o** [comment]
You can always go through the Github Sponsor page if you want to support my work:
https://github.com/sponsors/Rem0o

**Reply by RainerRocks** [reply]
OK sounds good,but I don't want my real name showing as a sponsor for the world to see  {I see some peoples real names showing as a sponsor}.

So after filling out the info on the sponsor page {I took a gander at it } do I get to choose what name shows up as a sponsor ? Thanks.

**Reply by Rem0o** [reply, marked as answer]
I believe your user name/profile name shows and not actually your real name. Some people have their real name as their username, must be what you saw.

**Comment by RainerRocks** [comment]
Ok I'll give it a go in a few, thanks.

**Comment by RainerRocks** [comment]
Hey buddy, I sent you $50 as a one time payment last Friday. Today my
credit card contacted me and said a new recurring payment for $50 has been
set up .
I told them not true ,I purposely picked a one time payment.

Please contact github ,them bastards causing me problems {not your fault} .
I called the number my credit card company gave me for github ,but it
directs you to their website,
no one actually picks up the phone. I sent them an email.

Thanks for your time.

**Reply by Rem0o** [reply]
HI,

Click your account icon at the top right, then click sponsors.  There should be a section to manage your sponsorships, and you can cancel any at any time.

![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/ab4f9cd2-b319-4b12-88b1-37649f16dcce)

**Reply by Rem0o** [reply]
Also, from my end, you are not a reccuring sponsor, just a one-time, as expected:
![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/69f39f5e-9946-4961-92f0-9c900b7d5c17)

**Comment by RainerRocks** [comment]
Hi Remo,

Yes I see that and it's a one time payment,even the receipt sent last Friday to my github email shows  it as a one time payment.

My CC company texted me { I get alerts] about 2 hours ago saying they noticed a new recurring charge posted to my account. So I called them and said its a one time payment and I even have the receipt .CC said it's showing as a recurring charge and gave me githubs phone number which directs you to their website ,so I created a ticket.

There's a mix up some where,not your fault, has to be the codes github is using that shows as a recurring charge to my CC .

**Reply by Rem0o** [reply]
My best guess is that the github payment system is made for reccuring, and 1 time is managed as a "reccuring", but expires immediatly so it only charges 1 time. I wouldn't expect any other charges.

**Reply by RainerRocks** [reply]
I hope so, weird thing is when I sent payment Friday I was alerted Friday of payment, no problem. Thing is, today is Monday and got the alert of recurring charge today ,not Friday. 

Ok I'll wait on githubs reply ,thanks for your quick replies, appreciated.

PS: none of this is your fault,just thought I contact you being I sent the money to you . Thanks again.

**Reply by Rem0o** [reply]
Alright keep me in the loop if anything weird happens, and thanks for your sponsorship, much appreciated!

**Comment by RainerRocks** [comment]
I will ,thanks.
